ar_transaction_changes.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency "activerecord", ">= 4.2.4", "< 6.0"
+  gem.add_dependency "activerecord", ">= 4.2.4", "< 7.0"
```

Comment:
> May I suggest removing the upperbound altogether?
> If we test against Rails HEAD there wouldn't be any problems going unnoticed due to not having an upperbound.

Yes, I support getting rid of the upper bound and testing against rails HEAD.

I'm actually surprised this dependency didn't cause a problem already for Shopify given that we are already on the rails 6 beta

Reply:
I think for Rails 6.0 > 6.0.0.beta

Reply:
oh right, so this would be a problem for us right after rails 6.0 is released

Reply:
@dylanahsmith to allow testing against Rails head do I just need to change [this line](https://github.com/dylanahsmith/ar_transaction_changes/pull/21/files#diff-de4cec2383a868c0317a72306723481cR3) to `gem "rails"`?

Reply:
For head we would need `gem 'rails', github: 'rails'`

`gem "rails"` would be latest release (currently 5.2.2). Which we might also want to test explicitly maybe. I think adding 2 Gemfiles that are `rails head` and `latest release` would make sense for easier future maintenance.

Reply:
This gem doesn't actually depend on rails, it just depends on active record, so I think we want `gem 'activerecord', github: 'rails/rails'` to test against rails HEAD

Reply:
Yes. I am not sure why `Gemfile.*` specifies rails version directly. I guess we could get away with `gem 'activerecord...` as well

Reply:
Adding only activerecord sound like a good idea
Added in the last commit